### DISCUSSION TITLE
Logs/Data Extraction SO

### DISCUSSION THREAD

**Discussion by Mav1814** [Security-Onion-Solutions/securityonion#11479] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32GB

### Storage for /

104G

### Storage for /nsm

206G

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hello! 
I hope you're all well!

I've been having trouble extracting the logs/data for the Security Onion Dashboard/Hunt fields. In other words, both in the SO and in kibana, the default fields are empty. At the moment, two agents are only sending logs. One of them I tried to perform intrusion attempts via ssh, and I only saw the information on the kibana side.

I therefore have a few questions.

1 - How do I extract the information I want from the logs to appear in the SO?
2 - How do I manipulate the default information in the logs, for example if I don't want all the event or host information to appear?

The goal is to implement your solution in my organization, but we have encountered some difficulties with the logs. We need to understand that part better. 

I hope I can explain my questions.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
What kind of endpoints are you ingesting logs from? Screenshots of your problem would be helpful in this case if possible.

**Reply by Mav1814** [reply]
Hi, I'm ingesting from windows10 and an ubuntu server. 
I've realized that a lot of the information in the logs can only be obtained by performing queries in the dashboard or hunt.

For example, to get the logs of failed ssh attempts, you need to create a query of the type "event.action:"logon-failed"" for windows. 

My question was whether it's possible to receive all types of logs in the security onion dashboard and not always have to create several queries to obtain information. Because what happens at the moment is that I always have to go to kibana to look for the logs.

**Reply by reyesj2** [reply]
Yes, you just need to go less granular in your search. The hunt interface is for granular searches like event.action:"logon-failed". If you want more of a high level overview for logs use dashboards. There you will find some preconfigured dashboards for you to view. You can also create your own custom dashboard. https://docs.securityonion.net/en/2.4/soc-customization.html?highlight=custom#custom-queries

Instead of a granular search like event.action maybe take a look at that particular logs event.category and different fields in those logs. That way you can create a custom dashboard that includes all the information you want on one page. Something like below that you can expand as you like

```
event.category: "host"  | groupby event.action
```

**Reply by Mav1814** [reply]
Hi @reyesj2 thank you for the reply

I'm starting to understand. 
In other words, in order to obtain specific information, specific queries have to be created. Right?

Otherwise, for example, if I want to see failed login attempts clearly, would it be best to create playbooks that generate this type of alert?

Or is it better to have specific queries and create new dashboards?

I don't know if I'm explaining myself well, I apologize.

**Reply by reyesj2** [reply, marked as answer]
Yes, you're understanding correctly.

Typically, if you're running the same query more than once or twice. You'll want to create an alert for that, so that you free up your time.

For an endpoint you're ingesting logs from, creating a playbook is a great choice for that alert. I recommend watching https://www.youtube.com/watch?v=SUXxlSlse_k and other YouTube videos on our channel.

**Reply by Mav1814** [reply]
Again, much thanks @reyesj2 . You helped me a lot to better understand this subject!! 

Cheers! See you around!!
